tools/RAiDER/utilFcns.py

```diff
@@ -165,20 +179,24 @@ def nodataToNan(inarr, listofvals):
     """
     Setting values to nan as needed
     """
+    inarr = inarr.astype(float) # nans cannot be integers (i.e. in DEM)
     for val in listofvals:
         if val is not None:
             inarr[inarr == val] = np.nan
 
 
 def rio_stats(fname, band=1, userNDV=None):
+    if os.path.basename(fname).startswith('S1-GUNW'):
+        fname = os.path.join(f'NETCDF:"{fname}":science/grids/data/unwrappedPhase')
+
     if os.path.exists(fname + '.vrt'):
         fname = fname + '.vrt'
 
     # Turn off PAM to avoid creating .aux.xml files
     with rasterio.Env(GDAL_PAM_ENABLED="NO"):
         with rasterio.open(fname) as src:
-            gt = src.transform.to_gdal()
-            proj = src.crs
+            gt    = src.transform.to_gdal()
```

Comment:
Seems like uneeded extra space?

Reply:
just a stylistic choice

I prefer (ok the idea is that tehy're all aligned in my script... markup is moving them here):
            gt     = src.transform.to_gdal()
            proj  = src.crs
            stats = src.statistics(band)
to 
            gt = src.transform.to_gdal()
            proj = src.crs
            stats = src.statistics(band)